src/css/profile/mobile/fonts.less

```diff
@@ -1,14 +1,18 @@
 @font-face {
     font-family: Roboto-Light;
```

Comment:
font-family name is different in guides
sans-serif-light
sec-roboto-light

Reply:
I think `sans-serif-light` and 'sec-roboto-light' are alternative font-families for `Roboto`.
I'll do changes. Thanks.

Reply:
I made changes. I made and used function `.font(style)`.
You can see this in file `tizen.less`